stable/rbac-manager/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: rbac-manager
-version: 0.1.0
```

Comment:
Had to bump this since the chart folder was changing.  Maybe there's a way around that.....